operator/v1/types_network.go

```diff
@@ -454,25 +454,56 @@ type NetFlowConfig struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// newCollectors is list of objects featuring a valid IPv{4|6} address and a port with a maximum of ten items
```

Comment:
```suggestion
	// newCollectors is list of objects featuring a valid IP address and a port with a maximum of ten items
```

"either IPv4 or IPv6" should just be assumed as the default these days.
(if we were going to be explicit, then say "IPv4 or IPv6", not "IPv{4|6}")

Also, "list of objects" is incredibly vague. What is this list of objects used for? Actually, the "collectors" field is mostly badly-documented as well. The `NetFlowConfig.Collectors` field is good, and you should use that as the basis for the docs for the new fields.

Reply:
Done.